lib/ExpensiMark.js

```diff
@@ -668,7 +700,7 @@ export default class ExpensiMark {
      *
      * @returns {String}
      */
-    replace(text, {filterRules = [], shouldEscapeText = true, shouldKeepRawInput = false, disabledRules = []} = {}) {
+    replace(text, {filterRules = [], shouldEscapeText = true, shouldKeepRawInput = false, disabledRules = [], extras} = {}) {
```

Comment:
Rename this new parameter to something a little more detailed like `extraReplacementFunctionParams` or `additionalReplacementFunctionParams`